src/components/Comments.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from "react";
+import { fetchCommentsByArticleId } from "./api";
+
+const Comments = ({ article_id }) => {
+  const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    fetchCommentsByArticleId(article_id).then((commentsData) => {
+      setComments(commentsData);
+      setIsLoading(false);
+    });
+  }, [article_id]);
```

Comment:
Because this component is rendering on the article page, this array has to happen, but it doesn't need the article id inside it. As the article page will re render if you've done the use effect correctly there's no need for the dependancy array to have an item in.